man/systemd-system.conf.xml

```diff
@@ -322,6 +322,11 @@
         The <literal>:</literal> means variables in the file can reference other environment
         variables, using <literal>${VARNAME}</literal> expansion.
         </para>
+
+        <para><varname>DefaultEnvironmentFile=</varname> defaults to empty for
+        system sessions, and defaults to <filename>/etc/environment</filename>,
+        <filename>/usr/lib/environment.d/*.env</filename>, and
+        <filename>~/.config/environment.d</filename> for user sessions.
```

Comment:
Like I wrote in the first comment, I hope that this setting can go away.

The list of dirs to look at should include /run/environment.d also, including documentation. It's possible that the actual (undocumented) list might need to be a bit longer, including /usr/local/lib/environment.d/, but not sure about that.

`*.conf` would be a better extension. The directory will not be used for other types of files, end `.env` look strange.